File: dongyimai_sellergoods_interface/src/main/java/com/offcn/sellergoods/service/BrandService.java
package com.offcn.sellergoods.service;

import com.offcn.entity.PageResult;
import com.offcn.pojo.TbBrand;

import java.util.List;
import java.util.Map;

public interface BrandService {

	//findAll
	public List<TbBrand> findAll();

	//findPage
	public PageResult findPage(int pageNum, int pageSize);

	//add
	public void add(TbBrand brand);

	//update
	public void update(TbBrand brand);

	//findOne
	public TbBrand findOne(Long id);

	//delete
	public void delete(Long[] ids);

	//条件、模糊查询+分页
	public PageResult findPage(TbBrand brand, int pageNum, int pageSize);

	/**
	 * 模板管理模块 查询品牌下拉列表
	 * @return
	 */
	public List<Map> selectOptionList();

}

File: dongyimai_manager_web/src/main/webapp/js/service/brandService.js
//品牌服务层
app.service('brandService',function($http){

	//findOne
	this.findOne=function(id){
		return $http.get('../brand/findOne.do?id='+id);
	}

	//add
	this.add=function(entity){
		return  $http.post('../brand/add.do',entity );
	}

	//update
	this.update=function(entity){
		return  $http.post('../brand/update.do',entity );
	}

	//dele
	this.dele=function(ids){
		return $http.get('../brand/delete.do?ids='+ids);
	}

	//模糊、条件查询+分页
	this.search=function(page,rows,searchEntity){
		return $http.post('../brand/search.do?page='+page+"&rows="+rows, searchEntity);
	}

	//模板管理模块 查询品牌下拉列表
	this.selectOptionList = function(){
		return $http.get('../brand/selectOptionList.do');
	}

});